.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: python
+
+python:
+    - "2.7"
+
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get install -y openjdk-8-jre-headless
+
+install:
+    - wget https://wiki.dnanexus.com/images/files/dx-toolkit-current-ubuntu-14.04-amd64.tar.gz
+    - tar xf dx-toolkit-current-ubuntu-14.04-amd64.tar.gz 
+    - source dx-toolkit/environment
+    - wget https://github.com/dnanexus-rnd/dxWDL/releases/download/0.46/dxWDL-0.46.jar
+
+script:
+    - dx login --token "$DX_TOKEN" --noprojects && dx select "$DX_PROJECT"
+    - date_stamp="$(TZ=":US/Pacific" date +"%Y-%b-%d-%H-%M-%S")"
+    - dx mkdir "$date_stamp"
+    - workflow_id=$(java -jar dxWDL-0.46.jar compile smartseq2_single_sample/ss2_single_sample.wdl -destination "$DX_PROJECT":/"$date_stamp")
+    - >
+        analysis_id=$(dx run "$workflow_id" --yes --folder "$DX_PROJECT":/"$date_stamp" --brief -istage_0.fastq_read1=file-F7FxyyQ03bypfYvp1QPqXP7Y
+        -istage_0.fastq_read2=file-F7Fxyyj03byQ4F4z1Q7jkJ5F
+        -istage_0.gtf=file-F7Fxyy803byQ4F4z1Q7jkJ59
+        -istage_0.ref_fasta=file-F7Fxyx803byV6xvf1PZyzv5z
+        -istage_0.rrna_interval=file-F7Fxyy803bykQjFB1Pj3FV4b
+        -istage_0.ref_flat=file-F7Fxyz003byYVBkP1PzZ9B87
+        -istage_0.star_genome=file-F7FxyzQ03bypbvg11V4ZkXGJ
+        -istage_0.output_prefix="smartseq2_portability_test"
+        -istage_0.rsem_genome=file-F7Fxyz803byQ4F4z1Q7jkJ5J)
+    - while [ $(dx describe "$analysis_id" --json | jq  -r .state) = in_progress ]; do echo "I'm not dead!"; sleep 30; done
+    - dx wait "$analysis_id"
+    - metrics_hash=$(dx download $(dx describe "$analysis_id" --json | jq -r '.output."stage_6.aln_metrics"."$dnanexus_link"') -o - | tail -n +6 | md5sum | awk '{print $1}')
```

Comment:
Do I understand correctly here - we're downloading the alignment metrics file and doing an md5 on the non header lines? Some comments about what this and other steps are doing would be helpful.

Reply:
Yeah that's right, I'll add a comment. The idea here is to show a simple case of the concerns a "checker" tool would have to address.